scripts/deploy_jenkins.sh

```diff
@@ -45,4 +45,8 @@ if [ "${NEXUS}" = true ]; then
    oc new-app -f  $TEMPLATES_DIR/nexus3-template.yaml
 fi
 
+if [ "${ENABLE_OAUTH}" = true ]
+  oc adm policy add-cluster-role-to-group view system:authenticated -n $PROJECT_NAME
```

Comment:
Also, this will require the user that you are logged in with to be an admin user before you can run this script. Would be better to at least allow the script to continue if you are not and output a warning.